docs/try/from-source.md

```diff
@@ -0,0 +1,58 @@
+# Running `witnet-rust` from source code
+
+## Install compilation dependencies
+
+### Rust 
+
+`witnet-rust` is developed using Rust programming language.
+You need to have it installed on your system to be able to compile the source code.
```

Comment:
```suggestion
You need to install Rust on your system to be able to compile the source code.
```

Reply:
"have it installed" also means ["ask or instruct someone to install it for you"](https://dictionary.cambridge.org/grammar/british-grammar/have-something-done) :wink:

Reply:
fixed! thanks